documentation/docs/configuration.md

```diff
@@ -14,10 +14,41 @@ Configuration of the Piper steps as well the Piper templates can be done in a hi
 2. Stage configuration parameters define a Jenkins pipeline stage dependent set of parameters (e.g. deployment options for the `Acceptance` stage)
 3. Step configuration defines how steps behave in general (e.g. step `cloudFoundryDeploy`)
 4. General configuration parameters define parameters which are available across step boundaries
-5. Default configuration comes with the Piper library and is always available
```

Comment:
I removed this line since this is a different dimension of the configuration framework. On the one hand have have parameters via signature followed by stage, step and general configuration. But step, stage and general repeats at several level. On the other hand have the project level, the custom level(s) and the default level. It is IMO better not to mix up stage, step, general with the layers (project, custom, default).

Here in this level we explain the signature, stage, step and default approach which it better not mixed up with the project, custom and default layer. The layers are now explained below.